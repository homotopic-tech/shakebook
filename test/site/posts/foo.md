---
title: The Foo In The Valley
posted: 2020-04-28
tags: [foo, spatula]
---

Lorem ipsum
