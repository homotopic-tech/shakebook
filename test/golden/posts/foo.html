<html>
  <head>
    <title>The Foo In The Valley</title>
  </head>
  <body>
    <h1>This post is called The Foo In The Valley</h1>
    <p>
      <small>Posted on Tuesday, April 28, 2020 with tags:
         <a href="/posts/tags/foo">foo</a>
         <a href="/posts/tags/spatula">spatula</a>
      </small> 
    <p>It was generated from the source posts/foo.md</p>
    <p>For the url /posts/foo.html</p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the recent posts again:
      <ul>
        <li>
          <a href="/posts/foo.html">The Quux In The Maw</a>
          <small>Posted on Monday, July 20, 2020 with tags:
              <a href="/posts/tags/quux">quux</a>
              <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
        <li>
          <a href="/posts/foo.html">The Baz In the Hills</a>
          <small>Posted on Sunday, June 28, 2020 with tags:
              <a href="/posts/tags/baz">baz</a>
              <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
        <li>
          <a href="/posts/foo.html">The Bar in the mountains</a>
          <small>Posted on Monday, June 01, 2020 with tags:
              <a href="/posts/tags/bar">bar</a>
              <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
      </ul>
    </p>
  </body>
</html>
